charts/drogue-cloud-examples/values.yaml

```diff
@@ -53,3 +53,19 @@ timescale:
     image: docker.io/library/postgres:14
   pusher:
     image: ghcr.io/drogue-iot/postgresql-pusher:0.1.0
+
+pusher:
+  enabled: true
+  knative: false
+  ingress: false
+  image: ghcr.io/drogue-iot/postgresql-pusher:0.1.0
```

Comment:
Couldn't we drop the `.timescale.pusher.image` then?

Reply:
Yeah, I forgot to remove it. Thanks